### PR TITLE
Feature/entity tree select

### PR DESCRIPTION
**Pull request by RajeevSiewnath**

### CODE REVIEW

**Comment by TehWazzard** [comment]
Would you be able to provide some usage examples in the `lib-tester` or in `usage.md`

**Comment by RajeevSiewnath** [comment]
@gbervik Right, it is setup in a way that you have to provide ALL entities you wish to have enabled (either as a flat array or nested with the children attribute of the entity). I could make a flag that controls whether children are always selectable. That would resolve that at least.

The entity tree component shows all entities. It uses the entity tree service to actually fetch which entities are selectable. Depending on the fetch mode it will fetch it automatically for you. The default fetch mode is DEFAULT, which basically fetches all entities based on the query input (which will filter down fetched entities when requesting entities from the API endpoint) and cache that. So if you for example would do something like: 

`<ws-entity-tree-select [entities]="[{id: 286}]" [query]="{member_of: 286}"></ws-entity-tree-select>`

It would only the tree of 286's children and make them selectable (after I woul dhave implemented that children of given entities are always selectable). 

I think adding some usage examples would make this a bit clearer as @TehWazzard mentioned

File: changelog.md [hunk 1]
```diff
@@ -1,5 +1,10 @@
 # Chaangelog
 
+## 8.7.0
+
+* userService.getUser
+* userService.getUserByPerson
```
Comment (fabian) [inline changelog.md:6]: Can you mention the new entity tree here?

Comment (fabian) [reply changelog.md:6]: Yeah I think it should get a new minor number but I'll leave the release process to @TehWazzard.

Ah didn't realize the meta key. Yes I can see how just Left/Right could be confusing. Just tried it and I think it's fine with the meta key - it's a power user feature after all!

Comment (TehWazzard) [reply changelog.md:6]: Yeah, I will do the version bump once we merge :)

Comment (TehWazzard) [reply changelog.md:6]: checking if this point is resolved. if yes I can start merging.

Comment (fabian) [reply changelog.md:6]: Changelog hasn't been updated yet but that shouldn't prevent this from being merged - can always be updated later.

However I'm still not clear on this:

> I'm still not sure how we would go about showing the full tree without any filtering - I would expect `<ws-entity-tree-select></ws-entity-tree-select>` but that turns up empty? (added in the tester now)

**Comment by RajeevSiewnath** [comment]
Sorry I’ll do that today. 

I was focussing on the user’s section of the people’s app. But I’ll make the changes for the library today

File: changelog.md (hunk 1 above)
Comment (RajeevSiewnath) [reply changelog.md:8]: When entities === null (by default) it shows all entities

**Comment by TehWazzard** [comment]
I will merge and release today.

**Pull request merged**